codespeed/templates/codespeed/timeline.html

```diff
@@ -3,6 +3,7 @@
 {% block title %}{{ block.super }}: Timeline{% endblock %}
 
 {% block extra_head %}
+{{ block.super }}
```

Comment:
If you override base_site.html in your site, and add something to `extra_head` that you want in timeline.html, it won't be added unless you include block.super. You have the same thing in the comparison.html template.